server/models/index.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose')
+
+
+//Define a schema
+const urlSchema = new mongoose.Schema({
+    date_created: Date,
+    originalURL: String,
+    shortenedURL: String
```

Comment:
"ShortenedURL" is called "shortURL" in other parts of the program. As a result, it's not being stored in the database. It otherwise works, and I have a working version of the code I can commit right now, or Megan, you can change it. I'm not sure what you want, and you are responsible for the code so I'll let you be, since now I'm not sure what anyone wants me to do.

Reply:
good call TK. I will change and add to the branch.